_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -204,7 +204,9 @@ version 0.20.*
   node will entirely use P2P address discovery without relying on the
   centralized DNS seeds.
 
-- [LND #4228][] walletrpc: add LabelTransaction endpoint to retrospectively label txns FIXME:dongcarl
+- [LND #4228][] adds a new wallet command `labeltx` for labelling onchain
+  transactions. This is a continuation of the work done in [LND #4213][] which
+  allowed setting a label option at send-initiation time.
```

Comment:
Perhaps remove the "option"? What do people think?

Reply:
+1 for removing "option"

Reply:
Done!